core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9110,10 +9115,26 @@ private void checkLiteral2(String expression, String expected) {
     final String expectedTrino = expectedPresto;
     final String expectedSpark = "SELECT MAP (ARRAY ('k1', 'k2'), ARRAY ('v1', 'v2'))\n"
         + "FROM (VALUES (0)) `t` (`ZERO`)";
+    final String expectedClickHouse = "SELECT map(array('k1', 'k2'), array('v1', 'v2'))";
     sql(query)
         .withPresto().ok(expectedPresto)
         .withTrino().ok(expectedTrino)
-        .withSpark().ok(expectedSpark);
+        .withSpark().ok(expectedSpark)
+        .withClickHouse().ok(expectedClickHouse);
+  }
+
+  @Test void testEmptyMapOrArrayValueConstructor() {
```

Comment:
these tests are not really useful, since they do not exercise your new changes.
but perhaps these tests cannot currently be written.
I apologize for the confusion. Let's delete them.

Reply:
Thanks，done @mihaibudiu